pyproject.toml

```diff
@@ -206,3 +205,6 @@ select = [
 "torch/_vendor/**" = [
     "UP", # No need to mess with _vendor
 ]
+
+[tool.ruff.lint.flake8-comprehensions]
+allow-dict-calls-with-keyword-arguments = true
```

Comment:
Nice feature, but we need to list this behavior deviation from flake8-comprehensions somewhere. (perhaps in a comment)

Reply:
Updated.